pkgs/amtool-caascad/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, runCommand
+, makeWrapper
+, prometheus-alertmanager
+, vault
+}:
+
+stdenv.mkDerivation rec {
+  pname = "amtool-caascad";
+  version = "1.0.0";
+
+  buildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+  dontConfigure = true;
+  unpackPhase = ":";
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${./amtool-caascad.sh} $out/bin/amtool-caascad
+    chmod +x $out/bin/amtool-caascad
+
+    substituteInPlace $out/bin/amtool-caascad --replace AMTOOL_CAASCAD_VERSION ${version}
+    wrapProgram $out/bin/amtool-caascad --prefix PATH ":" ${lib.makeBinPath [ vault ]}
```

Comment:
You can add `amtool` in the list to properly depend on it

Reply:
done